iroh/src/commands/doctor.rs

```diff
@@ -97,6 +98,14 @@ pub enum Commands {
         #[clap(long)]
         local_derper: bool,
     },
+    /// Attempt to get a port mapping to the given local port.
+    PortMap {
+        /// Local port to get a mapping.
+        local_port: NonZeroU16,
+        /// How long to wait for an external port to be ready.
```

Comment:
nit: `in seconds` or sth like that is missing.

Reply:
done in 552e1c7